spec/views/report/_report_list.html.haml_spec.rb

```diff
@@ -19,9 +19,16 @@
                                   :trees         => {:reports_tree => {:active_node => "xx-1_xx-1-0"}})
   end
 
-  it "Check links in the list view" do
-    render
-    expect(response).to have_selector("//tr[@onclick=\"miqTreeActivateNode('reports_tree','xx-1_xx-1-0_rep-5');\"]")
-    expect(response).to have_selector("//tr[@onclick=\"miqTreeActivateNode('reports_tree','xx-1_xx-1-0_rep-6');\"]")
+  context 'check links in the list view' do
+    let(:user) { FactoryBot.create(:user_with_group) }
+
+    before do
+      login_as user
+    end
+
+    it 'check links in the list view' do
+      render
+      expect(response).to have_selector("//div[@id=\"report_list_div\"]")
```

Comment:
This isn't exactly the same test.  This just checks that the report list is present, whereas the old test was looking for clickable links in the list view.  I think this needs to change to `'check links in the list view'`.

Reply:
I will change the test case header..